e2e/cypress/integration/messaging/inline_images_open_preview_window_spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            // # Visit a test channel
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+    it('M23345 - Inline markdown images open preview window', () => {
+        // # Enable 'Show markdown preview option in message input box' setting in Account Settings > Advanced
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#accountSettings').click();
```

Comment:
We're now following the recommended approach of `testing-library`. So, please use its [API queries](https://testing-library.com/docs/dom-testing-library/api-queries) as much as possible.
This could be,
```suggestion
        cy.findAllByLabelText('main menu').should('be.visible').click();
        cy.findByText('Account Settings').click();
```

Note on visibility check on first element (`main menu`). It's recommended to avoid flakiness since the page has been loaded from `cy.visit`.

Reply:
Fixed. I'll use these going forward.